lib/client.js

```diff
@@ -100,6 +106,19 @@ class Client {
 	// args.apiKey
 	// args.secretKey
 	makeRequest(args) {
+		// If we've had a request rejected because of rate limiting, we block here
+		// until our blocker has been lifted after a timeout.
+		if (this.rateLimitBlocked) {
```

Comment:
Much like the Vimeo Provider, I think that the request is fully dropped at this point.  It looks like any calls to this method are logged, then forgotten, and no queue is made.